code/modules/research/mechanic/mechanic_fabs.dm

```diff
@@ -65,7 +65,7 @@
 //We add the plastic cost to this output so people can see it
 /obj/machinery/r_n_d/fabricator/mechanic_fab/output_part_cost(var/datum/design/part)
 	var/output = ..()
-	if(plastic_added)
+	if(plastic_added && !(MAT_PLASTIC in part.materials))
```

Comment:
No, this is supposed to add the plastic cost to the output when things don't normally cost plastic.  Things that already have plastic in their cost get handled by `..()`.